deployment/dev/docker-compose.yml

```diff
@@ -31,12 +34,15 @@ services:
       - dev_app-network
       - monitoring_rzp-dev-monitoring
     ports:
-      - "8083:8083"
-      - "8084:8084"
+      - "8081:8081"
+      - "8082:8082"
```

Comment:
why change the ports here? With web and worker using the same ports, there would be conflict on local

Reply:
also check if we really need to expose these in worker? else drop

Reply:
So the `healthCheck` API runs on this port needed for pod health probes. Let me update the ports to avoid conflict with web.